index.html

```diff
@@ -1,37 +1,118 @@
 <!DOCTYPE html>
-
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>HTML & CSS Training</title>
-    <meta name="description" content="training" />
-    <meta name="author" content="Wolox" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" type="text/css" href="application.css" />
+    <title>HTML & CSS Training</title>
   </head>
-
   <body>
-    <div class="container">
-      <div class="container-img-book">
-        <img class="book-img" src="./assets/book-cover.png" alt="Book cover" />
-      </div>
-      <div class="container-book">
-        <div class="container-book-title">
-          <h1 class="book-title">Titulo del libro</h1>
-          <span class="description">(género)</span>
-        </div>
-        <div>
-          <h2 class="book-details">Autor del libro:</h2>
-          <span class="description">Nombre del autor del libro</span>
-        </div>
-        <div>
-          <h2 class="book-details">Editorial:</h2>
-          <span class="description">Nombre de la editorial</span>
-        </div>
-        <div>
-          <h2 class="book-details">Año de publicación:</h2>
-          <span class="description">Año publicación</span>
-        </div>
-      </div>
+    <nav class="menu">
+      <ul class="menu-list">
+        <li>
+          <a href="#" target="_self">
```

Comment:
You could add the image as a background image of the a tag you dont use the img tag